react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const root = document.querySelector('#root');
+
+ReactDom.render(CustomButton(), root);
```

Comment:
In react, we never really call our own components. Instead, create a React element.

```jsx
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```